graphql/schema.graphql

```diff
@@ -12,22 +12,15 @@ type Query {
     ### WORDS ###
     words: [Word!]! @all # All words
     word(literal: String!): Word @find # later use @search
-    #randomWord: Word # A random existing word
```

Comment:
Er du sikker på at du skal slette dem? :-)
Jeg har defineret disse kald jeg har brug for fra frontend. Så det er vist mening at man skal kunne fikse det sådan at man så kan kalde disse endepunkte, og ikke bare slette dem og *problem solved* :-P

Reply:
Jeg vil undgå at der ligger en masse ting som vi måske ikke komme til at bruge. Du kan bare notere hos dig.

Reply:
JEg kan ikke se på fronten/design at der er behov for randomWord, hvorfor jeg fjernede den.